ports/grimstorm/port.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": 2,
+  "name": "grimstorm.zip",
+  "items": [
+    "Grimstorm.sh",
+    "grimstorm"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Grimstorm",
+    "porter": [
+      "JanTrueno"
+    ],
+    "desc": "A place of terror and evil, spreading its foul corruption across the Kingdom. With it's malevolent ruler causing chaos and despair, it's time he met his demise. That's where you come in, heroic adventurer. Blessed with the gift to come back from death, it's up to you to storm the castle!",
+    "inst": "Download the windows version of the game here: https://gamejolt.com/games/grimstorm/59563\nPlace Grimstorm1.3.2.exe inside the gamedata folder.",
+    "genres": [
+      "action"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
```

Comment:
`gmloader` and `7zzs` are both `armhf` but you have flagged the port as `aarch64`.